assets/sip_samples/newspaper_tiff_alto_pdf_ebe47259-8f23-4a2d-bf49-55ae1d855393/data/mets.xml

```diff
@@ -88,19 +88,17 @@
                 DMDID="uuid-3936403d-133f-4765-b3b9-0a46df28db17 uuid-a4440db5-87f9-45af-819a-b966ca7f10fa"
                 AMDID="uuid-06efacfd-cc03-4e8f-b98a-e17b9e7eee1e"
             />
-            <div ID="uuid-60c6a853-b4fe-4658-ad8e-961adc15a00a" LABEL="Representations">
-                <div ID="uuid-5f92a639-0b45-4a9e-9c9e-e2a5a8764804" LABEL="representation_1">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_1/mets.xml" LOCTYPE="URL" />
-                </div>
-                <div ID="uuid-64055a8d-5f09-4cac-bb59-2726e3d624ff" LABEL="representation_2">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_2/mets.xml" LOCTYPE="URL" />
-                </div>
-                <div ID="uuid-b566f913-eff8-4a3d-84a4-fc982013bb8a" LABEL="representation_3">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_3/mets.xml" LOCTYPE="URL" />
-                </div>
+            <div ID="uuid-5f92a639-0b45-4a9e-9c9e-e2a5a8764804" LABEL="Representations/representation_1">
+                <mptr xlink:type="simple"
+                    xlink:href="./representations/representation_1/mets.xml" LOCTYPE="URL" xlink:title="representation_1" />
```

Comment:
`xlink:title` moet de waarde van de `@ID` van de filegrp van de representatie in de filesec hebben